bundles/org.openhab.binding.kaleidescape/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.8-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.kaleidescape</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Kaleidescape Binding</name>
+
+  <properties>
+    <dep.noembedding>commons-net,commons-lang3</dep.noembedding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>commons-net</groupId>
+      <artifactId>commons-net</artifactId>
+      <version>3.3</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.8.1</version>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
```

Comment:
It's planned to provide a replacement for the string escaping functions in the core of OH3. The plan is to change the Apache Commons invocations to core functions when OH3 is being released. So, these dependencies won't be necessary at all and should be removed, in contrary to my previous recommendation.

Reply:
@fwolter I removed the dependencies. What about SubnetUtils? I now see two warnings that the Apache libraries should not be used.....

Reply:
@cweitkamp There are 8 bindings, which use Apache Commons SubnetUtils. Will there be a replacement in OH3 core?

However, I saw that you used a fixed subnet mask of 24. That might work in most cases, but consider using https://docs.oracle.com/javase/8/docs/api/java/net/InterfaceAddress.html#getNetworkPrefixLength--

Reply:
> 
> However, I saw that you used a fixed subnet mask of 24. That might work in most cases, but consider using https://docs.oracle.com/javase/8/docs/api/java/net/InterfaceAddress.html#getNetworkPrefixLength--

Great catch! I fixed it so it is dynamic with getNetworkPrefixLength().

Reply:
@fwolter afaik OHC already has a [NetUtil](https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.core/src/main/java/org/openhab/core/net/NetUtil.java) class which could be use instead of `SubnetUtils`. Couldn't it? If not, which feature is missing?

Reply:
@cweitkamp This binding needs a list of all IP addresses in a given network address and network prefix.